app/consts.py

```diff
@@ -0,0 +1,2 @@
+sender = 'alidurrani550@gmail.com'
```

Comment:
pep8 issue. Please fix all of these issues from your code.

Reply:
sure, boss.